src/App/Pages/Accounts/LoginPasswordlessRequestPage.xaml.cs

```diff
@@ -21,6 +21,7 @@ public LoginPasswordlessRequestPage(string email, AuthRequestType authRequestTyp
             _vm.Page = this;
             _vm.Email = email;
             _vm.AuthRequestType = authRequestType;
+            _vm.PageTitle = _vm.HeaderTitle;
```

Comment:
⛏️ In order not to know the order in which these properties need to be set, you can move this to the `AuthRequestType` -> `set`, so after setting the backing field you can set the `PageTitle` and it reduces potential issues because of reordering properties assignation.